src/styles/styles.scss

```diff
@@ -82,10 +91,10 @@ body {
     text-decoration: none;
     margin-right: 20px;
     font-size: 20px;
```

Comment:
thoughts on having font-size variables for this font sizing?

Reply:
Ideally we just use the predefined type styles and never need to set font size individually. Down to create variables for this though and then use them within the text styles css definitions 👍🏽